bin/dsn.templates/csharp/single/csproj.template.php

```diff
@@ -108,7 +108,10 @@ function getGUID()
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="dsn.dev.csharp">
-        <HintPath><?=$dsn_root?>/lib/dsn.dev.csharp.dll</HintPath>
+        <HintPath><?=$dsn_root?>\lib\dsn.dev.csharp.dll</HintPath>
```

Comment:
Since dsn.csharp.dev has been compiled on Linux, I guess it could be handled by xbuild.
A still existed problem is that people now can't build rdsn until they install mono-xbuild. But if this is  a temporary branch used only by us, we can type "sudo apt-get install mono-xbuild -y"  before build.